### PR TITLE
[version-control] Switch default version control diff tool to git-gutter

### PR DESCRIPTION
**Pull request by sunlin7**
The default version control diff tool https://github.com/nonsequitur/git-gutter-plus and https://github.com/nonsequitur/git-gutter-fringe-plus are out of maintained for 6+ years, and has bugs for tramp (eg: https://github.com/nonsequitur/git-gutter-plus/pull/43 ).

Switch default VC diff tool to https://github.com/emacsorphanage/git-gutter for which is well maintained.

### CODE REVIEW

**Review by lebensterben — APPROVED** [review]

**Comment by sunlin7** [comment]
@lebensterben The document files are updated, please review again. Thanks.

**Review by lebensterben — APPROVED** [review]

**Comment by lebensterben** [comment]
I'm not familiar with the reason why git-gutter+ was selected as the default implementation.
So let's wait for approval(s) from more reviewers.

**Comment by smile13241324** [comment]
I have double checked the packages and @sunlin7 is right the current default is not maintained anymore. A quick test revealed no obvious loses in functionality in comparison with the non "+" implementations. With regards to future security fixes and adjustments to modern emacs versions I think it would be preferable to use the maintained packages by default.

Therefore I have merged the PR.

**Comment by sunlin7** [comment]
@smile13241324  Thank you !

**Pull request closed without merging**
